tokenserver/assignment/sqlnode/sql.py

```diff
@@ -475,6 +509,12 @@ def delete_user_record(self, service, uid):
         res = self._safe_execute(_DELETE_USER_RECORD, **params)
         res.close()
 
+    def migrate_user(self, service, uid, state):
+        """Set the migration state for a uid."""
+        params = {'service': service, 'uid': uid, 'state': state}
+        res = self._safe_execute(_MIGRATE_USER, **params)
+        res.close()
```

Comment:
I don't really understand the intention behind this "migration state" field. Do you have an example of how you might use it in practice?